docs/tutorials/iql.conf.yaml

```diff
@@ -0,0 +1,25 @@
+lessons:
```

Comment:
Probably worth a comment explaining what this file is. How were the values derived? When should I edit them?